tests/__init__.py

```diff
@@ -115,7 +115,9 @@ def assert_equals(frame1, frame2):
         assert frame1.stypes == frame2.stypes, (
             "The left frame has stypes %r, while the right has stypes %r"
             % (frame1.stypes, frame2.stypes))
-        assert frame1.to_list() == frame2.to_list()
+        assert frame1.to_list() == frame2.to_list(), (
+            "The frames have different data:\n"
+            "LHS = %r\nRHS = %r" % (frame1.to_list(), frame2.to_list()))
```

Comment:
Won't this produce too much of output if frames are different? For example for some tests with big data. May be we could have a parameter that controls verbosity of this function?

Reply:
Sure, but I mean the tests. I'm using this function for some moderate size data, guess I saw some of your large data tests, if test fails, this will produce a lot of noise :)

Reply:
And once a test fails and produces a lot of noise, there will be an incentive to go and fix the reporting routine to output a more meaningful diff. This is what I mean by "solve problems only as they appear". The current change is still an improvement: showing all the data is better than not showing any.